### PR TITLE
polish localisation

### PR DESCRIPTION
**Pull request by lesorubshayan**
close https://github.com/vim/vim/pull/10716

### CODE REVIEW

**Comment by ronin49** [comment]
@chrisbra 
it's using some complex script
my is just adding missing items

File: runtime/lang/menu_pl_pl.utf-8.vim [hunk 1]
```diff
@@ -1,8 +1,6 @@
 " Menu Translations:	Polish
-" Maintainer:		Mikolaj Machowski ( mikmach AT wp DOT pl )
-" Initial Translation:	Marcin Dalecki <martin@dalecki.de>
-" Last Change: 17 May  2010
-" Original translations
+" Maintainer:		Oleg Zadorozhnyi <lesorubshayan@gmail.com>
+" Last Change: 30 Apr 2024
```
Comment (k-takata) [inline runtime/lang/menu_pl_pl.utf-8.vim]: Did you explicitly take over the maintenance of this file from the former maintainer?
If yes, you can change "Maintainer" to your name, but you should keep the former maintainer's name as "Former Maintainer".

The "Original translations" line should be also kept.
`menu_pl_pl.iso_8859-2.vim` is automatically converted from `menu_pl_pl.utf-8.vim`, and this line shows whether the file is originally translated or automatically converted.

File: runtime/lang/menu_pl_pl.utf-8.vim [hunk 2]
```diff
@@ -291,3 +289,12 @@ let g:menutrans_fileformat_choices = "&Unix\n&Dos\n&Mac\n&Anuluj"
...
 let &cpo = s:keepcpo
 unlet s:keepcpo
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	Względna\ numeracja\ linii<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		Zawijanie\ linii<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		pakowanie\ słowem<Tab>:set\ lbr!
+menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			Tab\ rozszerzający\ się<Tab>:set\ et!
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		Automatyczne\ wcięcie<Tab>:set\ ai!
+menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin!		Wcięcie\ w\ stylu\ C<Tab>:set\ cin!
+menutrans Fold\ col&umn\ width			Złóż\ szerokość\ kolumny
+menutrans &Diff					Róż
```
Comment (girishji) [inline runtime/lang/menu_pl_pl.utf-8.vim]: Rozróżnij zamiast Róż.

File: runtime/lang/menu_pl_pl.utf-8.vim [hunk 3]
```diff
@@ -291,3 +289,12 @@ let g:menutrans_fileformat_choices = "&Unix\n&Dos\n&Mac\n&Anuluj"
 
 let &cpo = s:keepcpo
 unlet s:keepcpo
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	Względna\ numeracja\ linii<Tab>:set\ nru!
```
Comment (girishji) [inline runtime/lang/menu_pl_pl.utf-8.vim]: 'Toggle' to 'Przełącz'?

**Comment by ronin49** [comment]
@k-takata @girishji

File: runtime/lang/menu_pl_pl.utf-8.vim [hunk 4]
```diff
@@ -291,3 +290,12 @@ let g:menutrans_fileformat_choices = "&Unix\n&Dos\n&Mac\n&Anuluj"
...
 unlet s:keepcpo
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	Przełącz\ Względna\ numeracja\ linii<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		Przełącz\ Zawijanie\ linii<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		Przełącz\ pakowanie\ słowem<Tab>:set\ lbr!
+menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			Przełącz\ Tab\ rozszerzający\ się<Tab>:set\ et!
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		Przełącz\ Automatyczne\ wcięcie<Tab>:set\ ai!
+menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin!		Przełącz\ Wcięcie\ w\ stylu\ C<Tab>:set\ cin!
+menutrans Fold\ col&umn\ width			Złóż\ szerokość\ kolumny
+menutrans &Diff					Rozróżnij
+menutrans &Show\ File\ Types\ in\ menu		Pokaż\ typy\ plików\ w\ menu
```
Comment (chrisbra) [inline runtime/lang/menu_pl_pl.utf-8.vim:298]: Please move above line 291

**Comment by ronin49** [comment]
@chrisbra

**Comment by chrisbra** [comment]
@girishji do you ACK?

**Comment by chrisbra** [comment]
closing for the reasons mentioned here https://github.com/vim/vim/pull/14703#issuecomment-2092337585

**Pull request closed without merging**